src/types.rs

```diff
@@ -15,9 +15,27 @@ pub struct Measurement {
     pub humidity: Humidity,
 }
 
+/// A combined raw temperature / humidity measurement.
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
I'd also derive `Copy` since the type has just the size of an `u32`.

Reply:
Good point, I added it to all four measurement types.